src/Backend/AMQPBackendDispatcher.php

```diff
@@ -295,4 +295,24 @@ protected function getApiQueueStatus()
 
         return json_decode($request->send()->getBody(true), true);
     }
+
+    protected function resolveDelayStrategy(DelayStrategyAware $factory): DelayStrategyAware
```

Comment:
Should be `private` IMO

Reply:
Hey Oskar. 

To me this entire class is problematic. The class itself is not final so our project extended it in the past, ok so far. 

Then a static method was introduced with un-changeable settings being set. 

If I change the protected to private in my opinion we should go for the entire class to go final and let people define their entire class and user a compiler pass. 

So, I'd need a more direct feedback. Leave it or change it?